pibooth/booth.py

```diff
@@ -423,6 +423,9 @@ def main():
 
     parser = argparse.ArgumentParser(usage="%(prog)s [options]", description=pibooth.__doc__)
 
+    parser.add_argument("directory", nargs='?', default="~/.config/pibooth",
```

Comment:
I would change the option name to cfg_dir or something like that. My first thought when I hear directory is the pictures folder.

Reply:
done